### ISSUE TITLE
iOS 16.2 Notification don't work. Help :) 

### ISSUE THREAD

**Issue by AxopoulosDennis** [thudugala/Plugin.LocalNotification#391]
I'm using plugin.firebase in my project, I don't know if it has anything to do with that.
I commented usings and methods for firebase and I still could not receive local notifications.
With android it's working just fine. Am I missing something?

So I'm using your custom delegate
{
  `                
.UseLocalNotification(o =>
                  {
                      o.AddiOS(iOS =>
                      {
  #if IOS
                      iOS.UseCustomDelegate = true;
                      iOS.SetCustomUserNotificationCenterDelegate(new myproject.Platforms.iOS.CustomUserNotificationCenterDelegate());
  #endif
                      });
                  })
                  `
}

`    
public class CustomUserNotificationCenterDelegate : UserNotificationCenterDelegate
    {
        public override void DidReceiveNotificationResponse(UNUserNotificationCenter center,
            UNNotificationResponse response,
            Action completionHandler)
        {
            // if the Notification is type Plugin.LocalNotification.NotificationRequest
            // call the base method, else handel it by your self.

            var notificationRequest = TryGetDefaultIOsNotificationService().GetRequest(response.Notification.Request.Content);
            if (notificationRequest != null)
            {
                base.DidReceiveNotificationResponse(center, response, completionHandler);
            }
        }

        public override void WillPresentNotification(UNUserNotificationCenter center, UNNotification notification,
            Action<UNNotificationPresentationOptions> completionHandler)
        {
            // if the Notification is type Plugin.LocalNotification.NotificationRequest
            // call the base method, else handel it by your self.

            var notificationRequest = TryGetDefaultIOsNotificationService().GetRequest(notification?.Request.Content);

            if (notificationRequest != null)
            {
                base.WillPresentNotification(center, notification, completionHandler);
            }
        }
    }`

Then in my view

`    public readonly INotificationService _notificationService;

    public PlatformView(PlatformViewModel vm, INotificationService notificationService)
    {
        _notificationService = notificationService;
        InitializeComponent();
        BindingContext = vm;
        model = vm;

        this.Appearing += PlatformView_Appearing;
        this.MyWebView.Navigated += MyWebView_Navigated;
        this.MyWebView.Navigating += MyWebView_Navigating;
 }
`

    private async void PlatformView_Appearing(object sender, EventArgs e)
    {

        var notificationRequest = new NotificationRequest
        {
            BadgeNumber = 0,
            Description = "",
            Title = "You just lost first place!",
            ReturningData = "",
            NotificationId = 500,
            Schedule =
            {
                NotifyTime = DateTime.Now 
            },

        };

        await _notificationService.RequestNotificationPermission();
        await _notificationService.Show(notificationRequest);

}

**Comment by AxopoulosDennis** [comment]
I'm actually using Signal R and I want to notify user while the app is in  on sleep that there is a change to the channel through notification.

**Comment by AxopoulosDennis** [comment]
Hey guys, I found out what was the problem and I'm sharing it with you in case anyone comes across this.
Seems like I had to update info.plist and enable background modes. I enabled notifications and backround processing and now I'm receiving notifications!! Hooray. :)
![Screenshot 2023-03-21 at 1 33 05 PM](https://user-images.githubusercontent.com/122682590/226594342-637064ca-6463-4dac-9c11-65bac03c317f.png)
